specs/waku/v2/waku-relay-security-analysis.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Security Analysis of Waku Relay
+version: 
+status: Draft
+authors: Sanaz Taheri <sanaz@status.im>
+---
+
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Abstract](#abstract)
+  - [Terminologies](#terminologies)
+  - [Anonymity](#anonymity)
+    - [Definition](#definition)
+      - [Message Publisher Anonymity](#message-publisher-anonymity)
+      - [Topic Subscriber Anonymity](#topic-subscriber-anonymity)
+    - [Analysis](#analysis)
+  - [Authenticity](#authenticity)
+    - [Definition](#definition-1)
+    - [Analysis](#analysis-1)
+  - [Integrity](#integrity)
+    - [Definition](#definition-2)
+    - [Analysis](#analysis-2)
+  - [Confidentiality](#confidentiality)
+    - [Definition](#definition-3)
+    - [Analysis](#analysis-3)
```

Comment:
I'd probably make this a bit tighter.

Here are some examples that might give inspiration in terms of structure:

https://code.briarproject.org/briar/briar-spec/-/blob/master/protocols/BHP.md and https://code.briarproject.org/briar/briar-spec/-/blob/master/protocols/BTP.md

Reply:
Thanks! changed the structure